hw/ip/sram_ctrl/data/sram_ctrl_base_testplan.hjson

```diff
@@ -147,4 +147,40 @@
       tests: ["{name}_parity"]
     }
   ]
+
+  covergroups: [
+    {
+      name: key_seed_valid_cg
+      desc: '''
+            Covers SRAM receiving a key from OTP in Off/On states,
+            with both valid and invalid key seeds.
+            '''
+    }
+    {
+      name: lc_escalation_rst_cg
+      desc: '''
+            Covers de-assertion of LC escalation input before/after/during a reset trigger
+            to get SRAM out of terminal state.
+            '''
+    }
+    {
+      name: raw_hazard_cg
+      desc: '''
+            Covers RAW hazard occurrences in the SRAM, and also covers the occurrences that require
+            data forwarding (e.g. colliding accesses to same memory line).
+            '''
+    }
+    {
+      name: b2b_access_types_cg
+      desc: '''
+            Covers different of transaction types (read/write) that can create a b2b scenario.
```

Comment:
Covers different transaction types. Do we need the `of` here?

Reply:
ahh good catch, thanks!